main.py

```diff
@@ -1,7 +1,47 @@
-import init_django_orm  # noqa: F401
+import init_django_orm  # noqa F401
 
 from django.db.models import QuerySet
 
 
+from db.models import Genre, Actor
+
+
 def main() -> QuerySet:
     pass
+    genres_data = [("Western",), ("Action",), ("Drama",)]
+
+    for genre_name, in genres_data:
+        Genre.objects.create(name=genre_name)
+
+    actors_data = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for first_name, last_name in actors_data:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Drama").update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(
+        last_name="Clooney")
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue